EngineInterface/images/imapConnect.py

```diff
@@ -0,0 +1,115 @@
+import email
+import imaplib
+import os
+import smtplib
+import mimetypes
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.mime.base import MIMEBase
+
+def sendEmail():
+    fromaddr = 'smartron@cs.oswego.edu'
+    toaddrs  = 'noleary@oswego.edu'

```

Comment:
This shouldn't be hardcoded to one single email
```suggestion
    toaddrs  = toAddr
```